src/frontend/src/components/tables/InvenTreeTable.tsx

```diff
@@ -68,7 +68,7 @@ function loadActiveFilters(tableKey: string, filterList: TableFilter[]) {
  * @param filters : any - map of active filters, {name: value}
  */
 function saveActiveFilters(tableKey: string, filters: TableFilter[]) {
-  let active = {};
+  let active: any = {};
 
   filters.forEach((f) => (active[f.name] = f.value));
```

Comment:
You could simplify this to the following. Also, `any` types are not that great, in my project I decided to also disallow them strictly.
```suggestion
  const active = Object.fromEntries(filters.map((f) => [f.name, f.value]));
```